src/Agent.Worker/ExecutionContext.cs

```diff
@@ -421,6 +424,42 @@ public void InitializeJob(Pipelines.AgentJobRequestMessage message, Cancellation
                 Container = null;
             }
 
+            // Docker (Sidecar Containers)
+            if (message.JobSidecarContainers?.Count > 0)
+            {
+                SidecarContainers = new List<ContainerInfo>();
+                foreach (var sidecar in message.JobSidecarContainers)
```

Comment:
variable expand should live in JobRunner.cs as other variable expand.

Reply:
Moved variable expansion to JobRunner